api/src/pcapi/templates/extracts/beneficiary_extract.html

```diff
@@ -0,0 +1,194 @@
+                </g>
+            </g>
+            <rect y="64" width="4" height="531" transform="rotate(-90 0 64)" fill="#870087"/>
+            <defs>
+                <clipPath>
+                    <rect width="181" height="65" fill="white" transform="translate(382 60)"/>
+                </clipPath>
+            </defs>
+        </svg>
+        <h1>Réponse à ta demande d'accès</h1>
+        <div class="purple-background">
+            <p>
+                <i>
+                    Dans le cadre de l’utilisation des services de pass Culture nous sommes susceptible de collecter les données personnelles de nos utilisateurs, par exemple, pour assurer la gestion des réservations, adresser des bulletins d’actualité, lutter contre la fraude ou répondre à des demandes d’information. Le présent document te permet de prendre connaissance des données qui te concernent et qui sont utilisées pour le bon fonctionnement de nos services.
+                </i>
+            </p>
+            <p>
+                <i>
+                    Pour plus d'informations, tu peux également consulter notre <a href="https://pass.culture.fr/donnees-personnelles/">charte des données personnelles</a> ou contacter directement notre Délégué à la protection des données (DPO) : <a href="mailto:dpo@passculture.app">dpo@passculture.app</a>. 
+                </i>
+            </p>
+        </div>
+        <h3>📱 <span class=underline>Données de l’utilisateur</span></h3>
+        <table class="borderless">
+            <tr><td>Nom</td><td>{{container.internal.user.firstName}}</td></tr>
+            <tr><td>Prénom</td><td>{{container.internal.user.lastName}}</td></tr>
+            {% if container.internal.user.marriedName %}<tr><td>Nom de mariage</td><td>{{container.internal.user.marriedName}}</td></tr> {% endif %}
+            <tr><td>Adresse de messagerie: </td><td>{{container.internal.user.email}}</td></tr>
```

Comment:
Il n'y a pas de `:` sur les autres lignes.
```suggestion
            <tr><td>Adresse de messagerie</td><td>{{container.internal.user.email}}</td></tr>
```